ArmaForces.Boderator/ArmaForces.Boderator.BotService/Discord/DependecyInjection/DiscordServiceBuilder.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace ArmaForces.Boderator.BotService.Discord.DependencyInjection
+{
+    internal class DiscordServiceBuilder : IDiscordServiceBuilder
+    {
+        private IServiceCollection Services { get; }
```

Comment:
If it's private then it should not be a part of an interface I guess. It's not accessible outside anyway.

Reply:
I left this for future purposes like extra service configuration parameters.